electrum/plugins/convert2cc/qt.py

```diff
@@ -0,0 +1,236 @@
+You're about to convert one of your keystores/cosigners to Coldcard.<br>
+Please, close all other wallet windows. Make sure you have loaded the correct<br>
+wallet (do not forget BIP39 passphrase if used). If you have not connected your<br>
+Coldcard yet, please connect it (make sure no other app is using it) and close/re-open<br>
+this window. After this your Coldcard should match target keystores/cosigner<br>
+in this wallet. Your Coldcard does not need to be connected - but it is recommended.<br>
+<br>
+convert2cc does not rewrite your existing wallet file, but rather copy it<br>
+and create new one. After successful convert, new wallet will be opened.<br>
+</p>
+            """
+            non_cc_hw_keystores = [ks for ks in wallet.get_keystores() if self.hardware_keystore_not_coldcard(ks)]
+            vbox.addWidget(QLabel(_(description)))
+            ks_stack = QStackedWidget()
+
+            def select_ks(index):
+                ks_stack.setCurrentIndex(index)
+
+            labels = []
+            with self.coldcards_connected() as connected_cc_clients:
+                for idx, ks in enumerate(non_cc_hw_keystores):
+                    dev = self.match_candidate_keystore_to_connected_cc_device(connected_cc_clients, ks)
+                    if isinstance(wallet, Multisig_Wallet) and hasattr(ks, 'label'):
+                        res = _("cosigner") + f' {idx + 1}: {ks.get_type_text()} {ks.label}'
+                    else:
+                        res = _("keystore") + f' {idx + 1}: {ks.get_type_text()}' + f' {ks.label}' if hasattr(ks,'label') else ""
+                    if dev:
+                        res = res + 20 * " " + "[matches connected Coldcard]", self.paired_cc_icon_path
+                    else:
+                        res = res, self.unpaired_cc_icon_path
+                    labels.append(res)
+            # close opened cc devices --> check self.coldcards_connected
+
+            on_click = lambda clayout: select_ks(clayout.selected_index())
+            labels_clayout = ChoicesLayout(_("Select keystore"), labels, on_click)
+            vbox.addLayout(labels_clayout.layout())
+            btn_close = CloseButton(dialog)
+            btn_convert = QPushButton(_("convert"))
+            btn_convert.clicked.connect(lambda unused: self.do_convert2cc(main_window, labels_clayout, non_cc_hw_keystores, dialog))
+            vbox.addLayout(Buttons(btn_convert, btn_close))
+        vbox.addStretch(1)
+        dialog.setLayout(vbox)
+        dialog.exec_()
+
+    def do_convert2cc(self, main_window, labels_clayout, keystores, dialog):
+        wallet = main_window.wallet
+        selected_index = labels_clayout.selected_index()
+        if selected_index is not None:
+            target_keystore = keystores[selected_index]
+            with self.coldcards_connected() as connected_cc_clients:
+                # dev = self.match_candidate_keystore_to_connected_cc_device(connected_cc_clients, target_keystore)
```

Comment:
for now - not sending device to convert2cc - it is not needed and in current `ckcc-protocol` version it causes huge slowdown, see https://github.com/Coldcard/ckcc-protocol/pull/17